src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [username, setUsername] = useState(true);
+    const [password, setPassword] = useState(true);
+    const [responseMessage, setResponseMessage] = useState(null);
+    const [accessToken, setAccessToken] = useState(null);
+    const [accessExpiry, setAccessExpiry] = useState(null);
+    const [isAuthenticated, setIsAuthenticated] = useState(false);
+
+    const handleSubmit = async e => {
+        e.preventDefault();
+        let userInput = {}
+
+        new FormData(e.target).forEach((value, key) => {
+            if (key === "username")
+                userInput[key] = value;
+            else if (key === "password")
+                userInput[key] = value;
+
+        });
+
+
+        const payload = {
+            username: userInput["username"],
```

Comment:
userInput has a username and password that you are assigning to payload. Can you not use userInput, why make a copy of payload.

Reply:
fixed